src/GraphNodeManagerViewExtension/Controls/ControlColorsAndBrushes.xaml

```diff
@@ -0,0 +1,136 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <!-- Colors -->
+    <Color x:Key="Gray">#494949</Color>
+    <Color x:Key="MidLightGray">#5D5D5D</Color>
```

Comment:
These are already defined in DynamoColorsAndBrushes.xaml. Can we reuse them instead of defining again.

Reply:
Replaced with the closest values inside the DynamoColorsAndBrushes, I will try to flag that up to Helena and Jingyi to double-check if it looks alright to them.